_posts/2017-07-09-P3-Quality.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+asset-type: post
+name: P3 Quality
+title: 'P3 Quality'
+date: 2017-07-09 11:00:00 +00:00
+author: Raquel M Carmena
+image:
+  src: /assets/img/custom/blog/2017-07-09-P3-Quality.jpg
+tags:
+- craftsmanship
+- quality
+
+---
+
+What is software quality? Is it measurable? What should we focus on when we want to improve quality?
+
+It is still believed that quality in software is about automated functional tests. A manager worried about quality tends to contract external companies to automate functional tests.
+
+Standards ISO/IEC 250xx provide a lot of metrics and guides to measure the quality of a product, but is that enough? What about the process that is followed to create that product? 
+
+Others think that they must embrace agile methodologies, but a lot of agile adoptions fail, because they only focus on the process, no more.
+
+Quality is more complex and multi faceted.
+
+If we want to ensure or improve quality, we should focus on:
+
+* PROCESS
+* PRODUCT
+* PEOPLE
+
+So we could speak about **P3 QUALITY**:
+
+<center>
+<img src="{{site.baseurl}}/assets/img/custom/blog/2017-07-09-P3-Quality/P3-Quality.jpg" alt="P3 Quality" class="img img-responsive"/>
+<small>P3 QUALITY scheme</small>
+</center>
+
+<br/>
+
+The most important part is **PEOPLE**. When reading the book <a href="https://www.goodreads.com/book/show/23215733-the-software-craftsman" target="_blank">**The Software Craftsman. Professionalism, Pragmatism, Pride** by Sandro Mancuso</a>, you notice that the problems in software projects are the same, regardless of the company or the country, and the key to success is in the people’s quality, in our skills and attitude. 
+
+That’s the reason why <a href="http://manifesto.softwarecraftsmanship.org" target="_blank">**Software Craftsmanship**</a>, always trying to improve and raising the bar. Quality is in your hands.
```

Comment:
Thank you very much @Gryff. I've created another PR to fix it: https://github.com/codurance/site/pull/708